apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/SimpleManagedTracer.cs

```diff
@@ -171,7 +171,7 @@ public T RunInSpan<T>(Func<T> func, string name, StartSpanOptions options = null
                 catch (Exception e) when (SetStackTraceAndReturnFalse(e))
                 {
                     // This will never return as the condition above will always be false.
-                    return default;
+                    return default(T);
```

Comment:
Any reason for this change? Just `return default` should be fine, assuming you're using an up-to-date version of VS2017 (update 3).

Reply:
upgraded and fixed :)